README.md

```diff
@@ -26,7 +26,7 @@ You can install the current version of RWE-DataManagementModel as follows:
 
 ### Dependency `data.table` on mac
 
-While the `data.table` library should install automatically upon installation of the package, it may not function optimally through this installation process when installed on a mac (which will be the case for most users of this package).
+While the `data.table` library should install automatically upon package installation, it may not function optimally during this process when installed on a Mac (which will be the case for most users of this package).
```

Comment:
```suggestion
While the `data.table` library should install automatically upon package installation, it may not function optimally when installed on a Mac (which will be the case for most users of this package).
```